### ISSUE TITLE
Release of 1.1 / 2.0?

### ISSUE THREAD

**Issue by Josh-a-e** [facebook/react-360#689]
### Description

Hey, what are the plans to release react-360 1.1 or 2.0?

#### Expected behavior

Have the latest features

#### Actual behavior

Using the version that's approaching a year old.

### Reproduction

n/a

### Solution

Consolidate new features and cut a new release!  🎉 

### Additional Information

* 'react-360' package version: react-360@1.1.0
* 'react-360-web' package version: react-360-web@1.1.0 
* Operating System: Docker/Linux on MacOS
* Browser: Chrome 73.0.3683.103 (Official Build) (64-bit)
* VR Device: Cardboard via https://github.com/immersive-web/webvr-polyfill

**Comment by andrewimm** [comment]
1.1 was released months ago, and all documentation is updated to reflect it.

We are getting close to releasing an experimental version of 2.0, the contents of which are found in the `packages/` directory. Personally, I'm very excited for what we have in store. There will be more information about this coming soon, as there are just a few loose ends to tie up before it can go to beta testing.

**Comment by Josh-a-e** [comment]
Awesome, thanks a lot Andrew.

Apologies for not getting the version wrong, I was using [github releases](https://github.com/facebook/react-360/releases) as a point of reference rather than the [npm releases](https://www.npmjs.com/package/react-360).

Cool, yeah I noticed a couple of the examples weren't working and required a couple of imports from the webgl-ui library in there that I stumbled across (namely VRMath).

I'm going to assume you're not going to be able to specify any sort of specific date, regarding "soon"?
